cellmlmanip/model.py

```diff
@@ -458,12 +458,12 @@ def get_free_variable_symbol(self):
         # This should be unreachable
         raise ValueError('No free variable set in model.')  # pragma: no cover
 
-    def get_symbol_by_cmeta_id(self, cmeta_id):
-        """Searches the given graph and returns the symbol for the variable with the
-        given cmeta_id.
+    def _get_symbol_by_cmeta_id(self, cmeta_id):
+        """Searches the given graph and returns the symbol for the variable with the given cmeta_id.
+        PLEASE NOTE this does NOT get the oxmeta tag to get that use
+        get_symbol_by_ontology_term("https://chaste.comlab.ox.ac.uk/cellml/ns/oxford-metadata",
```

Comment:
Shouldn't the first argument here end in a `#`?

Reply:
Ah, I see `rdf.py` has a check for a missing `#` and adds it to avoid the most common user error! But our reference examples should definitely have it - the canonical URL for the ontology base ends in a `#`.

Reply:
I didn't touch this, just changed the function name

Reply:
You added the extra content in the docstring, and that's where the extra `#` is wanted.